### PR TITLE
KCD 2020 Campinas Cancellation

### PR DESCRIPTION
**Pull request by pasimoes**
Unfortunately, the effects of CONVID-19 were terrible and we could not postpone the event, the only alternative is the cancellation. 

The organizers are very sad about this but it is the best to do. Forgive us.

In our opinion, we need to maintain for 1 month the site as a reference to communication. After that, I suppose that it can be removed.

### CODE REVIEW

**Comment by KTan1226** [comment]
Thank you @pasimoes. Can you add a LGTM from another one of your organizers? I can merge after that.

**Comment by carolfvb** [comment]
I agree to this PR canceling the event.. LGTM 

LGTM is an acronym meaning looks good to me.

**Comment by KTan1226** [comment]
Hi @pasimoes- thanks for the updates. Can you please resolve the conflicts in this PR?

File: content/events/2020-campinas/_index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: Campinas - CANCELAMENTO
+date: '2020-04-04'
```
Comment (KTan1226) [inline content/events/2020-campinas/_index.md]: Please remove the date

**Comment by KTan1226** [comment]
Closing this PR as the event has been cancelled and passed. Thanks!

**Pull request closed without merging**